yt_dlp/utils/_utils.py

```diff
@@ -2575,6 +2575,13 @@ def url_or_none(url):
     return url if re.match(r'^(?:(?:https?|rt(?:m(?:pt?[es]?|fp)|sp[su]?)|mms|ftps?):)?//', url) else None
 
 
+def sum_or_none(*vals):
+    if None in vals:
+        return None
+    else:
+        return sum(vals)
```

Comment:
Just do `try_get(vals, sum)`. Not worth a function

Reply:
The function is now redundant since `traverse_obj` in the above will already `try_call`